src/main/resources/META-INF/mods.toml

```diff
@@ -73,7 +73,7 @@ MineColonies is a colony simulator within Minecraft! There are numerous types of
     side="BOTH"
 [[dependencies.minecolonies]]
     modId="multipiston"
-    mandatory=true
+    mandatory=false
```

Comment:
Like to have this mandatory for people still

Reply:
Why is this even an issue.
We are publishing this properly...

Reply:
Cuz it's not hard dep (no code ref), not sure if it's used in schematics tho

Reply:
I had it hard dep because its used in schematics

Reply:
That seems like a good reason to keep it mandatory.  Unless we want to rescan them to populate the `required_mods` and have the build tool warn you if a schematic needs a mod you don't have, but then that'd be a different PR anyway...